app.json

```diff
@@ -185,6 +185,10 @@
       "description": "If false, disables the node_modules cache to fix yarn install",
       "value": "false"
     },
+    "OAUTH_MAINTENANCE_MODE": {
+      "description": "If true, on login redirects to a static page with message",
+      "value": "false"
```

Comment:
I think you need to uppercase this to `"True"` and add `"required": false` as well.